pkg_resources/extern/__init__.py

```diff
@@ -70,12 +70,21 @@ def install(self):
             sys.meta_path.append(self)
 
 
+# [[[cog
+# import cog
+# from tools.vendored import yield_root_package
+# names = "\n".join(f"    {x!r}," for x in yield_root_package('pkg_resources'))
+# cog.outl(f"names = (\n{names}\n)")
+# ]]]
 names = (
     'packaging',
     'platformdirs',
+    'typing_extensions',
```

Comment:
This was not listed. The `vendored.txt` file says the following:

```
# required for platformdirs on Python < 3.8
typing_extensions==4.4.0
```

But now we removed support to Python < 3.8, so I guess we can remove this vendored dependency?